src/components/common/landingPage/categories/categories.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable react/forbid-prop-types */
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { getTags, getArticlesByCategory } from '../../../../redux/actions/landingPageActions';
+import Cards from '../card/cards';
+import './categories.scss';
+
+const Category = (props) => {
+  const [tagsIndex, setTagsIndex] = useState(0);
+  const [isFetching, setIsFetching] = useState(false);
+
+  const fetchArticles = async (allTags) => {
+    await props.getArticlesByCategory(allTags, tagsIndex);
+    setTagsIndex(tagsIndex + 10);
+    setIsFetching(false);
```

Comment:
`isFetching` is `false` by default and you are setting it to `false` here. I don't see where it was set to `true` though

Reply:
It is been set to true In the handleScroll() function.